HCIDOCS-651: Oracle Cloud C3 (#88897)
openshift/openshift-docs: modules/c3-assisted-installer-running-script-via-region.adoc

// Module included in the following assemblies:
//
// * installing/installing_oci/installing-c3-assisted-installer.adoc

:_mod-docs-content-type: PROCEDURE
[id="c3-ai-running-script-via-region_{context}"]
= Running the Terraform script via the C3 region

Create all infrastructure resources on the {oci-c3-short} system, including LBs, VCNs, and Instances. 

> During this procedure, you deploy an initial set of nodes, consisting of three control plane (master) nodes and three compute nodes. You then rename and reboot the nodes. This process results in temporary node duplication, requiring manual cleanup in the next procedure.

Reviewer: ```suggestion
During this procedure, you deploy an initial set of nodes, which consists of three control plane (master) nodes and three compute nodes. You then rename and reboot the nodes. This process results in temporary node duplication, requiring manual cleanup in the next procedure.
```

.Procedure

. Connect to the bastion server via SSH.

. Set the C3 Certificate location and export the certificate. 

> . Apply the terraform script to create three control plane nodes and three compute nodes. 

Reviewer: ```suggestion
. Apply the Terraform script to create three control plane nodes and three compute nodes. 
```

. Update the labels for the control plane and compute nodes.

> . Stop and start the instances one by one on the {oci-c3-short} portal.

Reviewer: ```suggestion
. Stop and restart the instances one by one on the {oci-c3-short} portal.
```
Should "start" be "restart"?

> For the full procedure, see link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Terraform Script Execution - Part 2 (Oracle documentation)].

Reviewer: ```suggestion
For the full procedure, see the "Terraform Script Execution Part-2 (Run Script via C3 Region)" section in the link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Oracle documentation].
```
